content/docs/tasks/telemetry/export-logs-through-stackdriver.md

```diff
@@ -0,0 +1,301 @@
+---
+title: Exporting Istio Access Logs to BigQuery, GCS, Pub/Sub through Stackdriver
+description: How to export Istio Access Logs to different sinks like BigQuery, GCS, Pub/Sub through Stackdriver.
+weight: 70
+---
+
+This task shows how to export Istio Access Logs to different sinks like
+BigQuery, GCS, Pub/Sub through Stackdriver A user can do analytics on data
+through Istio using logs flowing from Istio to StackDriver and then exporting
+these logs to various configured sinks like BigQuery, GCS or Pub/Sub.
+
+The [Bookinfo](/docs/guides/bookinfo/) sample application is used as the example
+application throughout this task.
+
+## Before you begin
+
+[Install Istio](/docs/setup/) in your cluster and deploy an application.
+
+## Configuring Istio for exporting logs to various sinks
+
+Istio exports logs through logentry template configured for Mixer as [accesslog
+entry](https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/charts/mixer/templates/config.yaml#L134:9).
+This specifies all the variables that would be available for analysis. It
+contains information like source service, destination service, auth
```

Comment:
suggestion: remove the `metrics(coming..)` and say something like 'as well source and destination user principals, among others'.